pilot/lib/pilot/build_manager.rb

```diff
@@ -215,16 +215,27 @@ def should_update_localized_build_information?(options)
     # If itc_provider was explicitly specified, use it.
     # If there are multiple teams, infer the provider from the selected team name.
     # If there are fewer than two teams, don't infer the provider.
+    # duplicated in deliver\lib\deliver\runner.rb
```

Comment:
Why is this duplicated? Can it be extracted into a static method somewhere?

Reply:
Was that way in the code, but as I only reopened a PR for changes that @Ruenzuo did I only added this note so this relationship becomes a bit more obvious. One could of course create an issue so someone else can fix this in the future. (Meaning: I think out of scope for this PR)